schemas/Error.json

```diff
@@ -3,17 +3,20 @@
   "description": "Triggered when an error occurred",
   "properties": {
     "eventName": {
+      "title": "EventNameError",
```

Comment:
Why not using "Error" as a prefix like for `ErrorDomain` and `ErrorName`?

Reply:
Updated fwiw.